ci/build.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: latest
```

Comment:
I think, we are supposed to specify an exact version of the golang repo to use rather than using the latest?
https://onswebsite.slack.com/archives/CCJMKPHPW/p1611165496070000

Same with the `Unit` file